digitalearthau/stacker.py

```diff
@@ -0,0 +1,242 @@
+"""
+A DEA version of `stacker` from ODC
+
+Uses the exact same implementation, but wraps it in QSUB Running/Event logging goodness.
+
+As such, it is a three step process, with the second two happening automatically after the first:
+
+1. Submit
+
+ Submit a single threaded PBS job to calculate how big the requested job is.
+ This step executes very quickly, creates the 'job' directory which groups log files
+ and metadata for the full job, then finally schedules a 'generate' job to run using
+ PBS qsub.
+
+2. Generate
+
+ Run as a single threaded process inside PBS to:
+ calculate how much work is required,
+ write the tasks out to a task file and then
+ schedule another PBS job of an appropriate size to 'Run' the stacking
+
+3. Run
+
+ Do all of the stacking. Spin up a Redis queue and use celery to farm the tasks
+ out across a large PBS job.
+"""
+import logging
+from datetime import datetime
+from functools import partial
+from math import ceil
+from pathlib import Path
+from typing import Tuple
+
+import click
+
+import datacube
+import digitalearthau
+from datacube.api.query import Query
+from datacube.index._api import Index
+from datacube.ui import click as ui
+from datacube.ui import task_app
+from datacube_apps.stacker import stacker
+from digitalearthau import __version__
+from digitalearthau import paths, serialise
+# pylint: disable=invalid-name
```

Comment:
What name was this disabling? Still necessary?

Reply:
Probably not, I'll disable it and see if anything complains.
Was probably for a click parameter that I stopped using.